cmd/kube-controller-manager/app/controllermanager.go

```diff
@@ -77,7 +77,7 @@ func NewCMServer() *CMServer {
 		NodeMilliCPU:            1000,
 		NodeMemory:              resource.MustParse("3Gi"),
 		SyncNodeList:            true,
-		SyncNodeStatus:          false,
+		SyncNodeStatus:          true,
```

Comment:
Yep, that will do exactly what I was trying to accomplish.  I'll re-push and remove the node controller updating flag